src/mcpbr/harnesses.py

```
@@ -555,6 +555,48 @@ def __init__(
+            return
+
+        setup_cmd = self.mcp_server.get_setup_command_for_workdir(env.workdir)
+        setup_timeout = int(self.mcp_server.setup_timeout_ms / 1000)
+
+        if verbose:
+            self._console.print(
+                f"[cyan]Running setup command (timeout: {setup_timeout:.0f}s)...[/cyan]"
+            )
+
+        # Source the env file so setup_command has access to API keys etc.
+        env_file = "/tmp/.mcpbr_env.sh"
```

Ruff (0.14.14)

[error] Probable insecure usage of temporary file or directory: "/tmp/.mcpbr_env.sh"

(S108)